src/org/omegat/filters2/text/bundles/ResourceBundleFilter.java

```diff
@@ -134,15 +137,45 @@ public boolean isTargetEncodingVariable() {
     }
 
     /**
-     * The default encoding is OConsts.ASCII
+     * The source default encoding is UTF-8.
+     * <p>
+     *     From Java 9 onward, properties are saved in UTF-8.
+     * </p>
     */
     @Override
     public Instance[] getDefaultInstances() {
-        return new Instance[] { new Instance("*.properties", StandardCharsets.US_ASCII.name(),
-                StandardCharsets.US_ASCII.name(), TFP_NAMEONLY + "_"
+        return new Instance[] { new Instance("*.properties", DEFAULT_SOURCE_ENCODING,
+                DEFAULT_TARGET_ENCODING, TFP_NAMEONLY + "_"
                 + TFP_TARGET_LOCALE + "." + TFP_EXTENSION) };
     }
 
+    /**
+     * Creates a reader of an input file.
+     * <p>
+     *     Override because of keep buggy behavior in OmegaT 5.7.1 or before.
+     *     It set default encoding US-ASCII but Java standard InputStreamReader
+     *     class wrongly accept non-ASCII characters as-is.
+     * </p>
+     *
+     * @param inFile
+     *            The source file.
+     * @param inEncoding
+     *            Encoding of the input file, if the filter supports it. Otherwise null.
+     * @return The reader for the source file
+     * @throws IOException
+     *             If any I/O Error occurs upon reader creation
+     */
+     @Override
+    public BufferedReader createReader(File inFile, String inEncoding) throws IOException {
+        Charset charset;
+        if (inEncoding != null) {
+            charset = Charset.forName(inEncoding);
+        } else {
+            charset = Charset.defaultCharset();
+        }
+        return new BufferedReader(new InputStreamReader(Files.newInputStream(inFile.toPath()), charset));
+    }
+
```

Comment:
### Workaround 1.

Override createReader() with ancient IO subsystem. to accept UTF-8 characters even encoding is specified as US-ASCII that is default on OmegaT 5.7.1 or before.